libs/ledger-live-common/src/wallet-api/react.ts

```diff
@@ -355,7 +355,16 @@ export function useWalletAPIServer({
           currencies: currencyList,
           onSuccess: (account: AccountLike, parentAccount: Account | undefined) => {
             tracking.requestAccountSuccess(manifest);
-            resolve(accountToWalletAPIAccount(account, parentAccount));
+            const walletApiAccount: WalletAPIAccount = accountToWalletAPIAccount(
+              account,
+              parentAccount,
+            );
+            if (parentAccount) {
+              // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+              // @ts-ignore
```

Comment:
Always prefer `@ts-expect-error` to `@ts-ignore` to avoid mask errors when not present and being explicit on what error is expected and why
```suggestion
              // @ts-expect-error Explain why
```

Reply:
FYI here ts-expect-error breaks if using the upcoming wallet-api as there would be no error. This is why I didn't follow your requested changes recommendations